### ISSUE TITLE
Verbose logging enabled in RunCommandInternalRaw

### ISSUE THREAD

**Issue by ccarvalheiro** [SRombauts/UE4GitPlugin#106]
In a recent commit, the logging in RunCommandInternalRaw was enabled by commenting `#if UE_BUILD_DEBUG`. See [here](https://github.com/SRombauts/UE4GitPlugin/blob/master/Source/GitSourceControl/Private/GitSourceControlUtils.cpp#L105
) and [here](https://github.com/SRombauts/UE4GitPlugin/blob/master/Source/GitSourceControl/Private/GitSourceControlUtils.cpp#L135).

Was this change made intentionally? It appears to slows down our mid sized projects and sometimes floods the log making it difficult to use.

**Comment by SRombauts** [comment]
Yes, it's intentional since I  received bug reports when I have only the logs to try to understand what's going on.

Sorry for the inconvenience, the idea would be to add a setting to have this optional.

**Comment by ccarvalheiro** [comment]
Oh, that makes sense. I can help with that. Should we add a checkbox to the source control dialog or hide it away in project settings (in the plugin settings category)?

**Comment by SRombauts** [comment]
Hello, until now I have always put everything on the source control dialog.

Perhaps should we also be able to find some of these settings also in the project settings? I never thought about it but I am pretty sure it would help people find them.

Cheers!
